pkg/kudoctl/kudoinit/migration/01_migrate_ov_names/migrate_ov_names.go

```diff
@@ -0,0 +1,124 @@
+//nolint:golint,stylecheck
+package _01_migrate_ov_names
```

Comment:
I welcome ideas for better package names. 
- I think it makes sense to have a package per migration
- It makes sense to have them ordered in some form - Not sure if it needs to be with numbers or something else